test/display.test.ts

```diff
@@ -122,4 +122,21 @@ describe('display', () => {
     );
     expect(stripAnsi(actual)).toEqual(stripAnsi(expected));
   });
+
+  it('should show test path in output if path present', async () => {
+    const { scanResults } = await scan({ path: helloWorldPath });
+    const errors: string[] = [];
+    const options = { path: '/path/to/project' };
```

Comment:
what's the difference between path and the path used on line 127

Reply:
This is to get the test, where we check the CLI prints the test path, to pass:
`helloWorldPath`, used on line 127, is defined earlier in `const helloWorldPath = path.join('test', 'fixtures', 'hello-world');` - this works to read our fixture, but not for testing the target path printed by the CLI.
Problem is that we can't check a concatenated path against a predefined string, because on Windows the path is joined with a '\' instead of a '/', so we got broken Circle tests. Since I haven't yet found an alternative/solution, I just passed in a single string as our path to avoid this problem.